### DISCUSSION TITLE
Cutting off the cloud improve the stream?

### DISCUSSION THREAD

**Discussion by guarddog13** [gtxaspec/wz_mini_hacks#297] [Q&A]
Does cutting these cameras off from the Wyze cloud after installing this hack help the reliability of them? The high res streams struggle quite a bit even on cameras in an arms reach of the router. 

My only issue with cutting them off from the cloud is losing microphone support. Agent DVR at least doesn't support the microphone. I believe tiny camera did but don't recall currently.

**Comment by bjs-pdx** [comment]
I could be wrong, but they are running the Wyze firmware plus the hack, so I don't think the cameras will bootup and run without an internet connection.
